### PR TITLE
10534 design debt

### PR DESCRIPTION
**Pull request by btejha**

### CODE REVIEW

File: package-lock.json
Comment (jimlerza) [inline package-lock.json:1]: I don't think we want changes to this file in this PR.

File: tmp.md
Comment (jimlerza) [inline tmp.md:1]: Please remove this file as well.

**Review by jimlerza — CHANGES_REQUESTED** [review]

File: tmp.md
Comment (btejha) [reply tmp.md:1]: oops forgot to removed it, done.

File: package-lock.json
Comment (btejha) [reply package-lock.json:1]: Understood.

**Review by pixiwyn — APPROVED** [review]

**Review by jimlerza — APPROVED** [review]

**Pull request merged**